tests/compiler/test_compile_export.py

```diff
@@ -96,17 +96,43 @@ def test_export_compile_data_to_file_error(snippetcompiler, tempfile_export) ->
 
 
 @pytest.mark.parametrize(
-    "snippet,exception,category",
+    "snippet,exception,category,message,report_exnc",
     [
-        ("1 = 1", ParserException, ast_export.ErrorCategory.parser),
-        ("x.n = 1", NotFoundException, ast_export.ErrorCategory.runtime),
-        ("import tests tests::raise_exception('my message')", ExplicitPluginException, ast_export.ErrorCategory.plugin),
+        (
+            "1 = 1",
+            ParserException,
+            ast_export.ErrorCategory.parser,
+            "Syntax error at token 1",
+            "inmanta.parser.ParserException",
+        ),
+        (
+            "x.n = 1",
+            NotFoundException,
+            ast_export.ErrorCategory.runtime,
+            "variable x not found",
+            "inmanta.ast.NotFoundException",
+        ),
+        (
+            "import tests tests::raise_exception('my message')",
+            ExplicitPluginException,
+            ast_export.ErrorCategory.plugin,
+            "my message",
+            "inmanta.plugins.PluginException",
```

Comment:
To fully test the new functionality here it might be good to update the plugin to raise a custom exception (deriving from `PluginException` and test that that type gets correctly output.

Reply:
done

Reply:
Shouldn't this be `"inmanta_plugins.tests.TestPluginException"` then?

Reply:
I see I replied too quickly.